src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/import.py

```diff
@@ -39,50 +36,42 @@ def acr_import(cmd,
         raise CLIError(INVALID_SOURCE_IMAGE)
     source_image = source
 
-    slash = source.find('/')
-
     ImportImageParameters, ImportSource, ImportMode = cmd.get_models(
         'ImportImageParameters', 'ImportSource', 'ImportMode')
-    if slash < 0:
-        if not source_registry:
-            from knack.prompting import prompt, NoTTYException
-            try:
-                source_registry = prompt(SOURCE_REGISTRY_MISSING)
-            except NoTTYException:
-                raise CLIError(NO_TTY_ERROR)
-        if not is_valid_resource_id(source_registry):
+
+    if source_registry:
```

Comment:
Consider clarifying in https://github.com/Azure/azure-cli/blob/fc7e28991c81d96d960c7d09cdf6c73cd81aae00/src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py#L76
to something like if `-r` is specified, `--source` is taken as just the image name; otherwise, `--source` is taken as the fully qualified name.